simpler_env/policies/openvla/openvla_model.py

```diff
@@ -0,0 +1,203 @@
+from collections import deque
+from typing import Optional, Sequence
+import os
+import matplotlib.pyplot as plt
+import numpy as np
+from transforms3d.euler import euler2axangle
+from simpler_env.utils.action.action_ensemble import ActionEnsembler
+from transformers import AutoModelForVision2Seq, AutoProcessor
+from PIL import Image
+import torch
+import cv2 as cv
+
+
+class OpenVALInference:
```

Comment:
OpenVALInference -> OpenVLAInference    ?

Reply:
This code is outdated.